packages/multichain-transactions-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0]
```

Comment:
This could have a been just a patch bump, but this is controller is not yet "stable", so that's probably ok. 😅